azure_functions_worker/dispatcher.py

```diff
@@ -523,7 +535,19 @@ async def _handle__invocation_request(self, request):
                     shmem_mgr=self._shmem_mgr,
                     is_deferred_binding=pb_type_info.deferred_bindings_enabled)
 
-            fi_context = self._get_context(invoc_request, fi.name, fi.directory)
+            http_v2_enabled = self._functions.get_function(function_id) \
```

Comment:
Why do this repeatedly for every invocation call? Can't it be done within the init or indexing itself?

Reply:
okay - looking below does make sense the need for computing this

Reply:
is_http_func which is a bool value is already set in indexing function time in the self._functions(functionregistry), this is to retrieve that attribute instead of determining if function is http func every time.